terraform/Makefile

```diff
@@ -37,5 +38,18 @@ plan_main:
 apply_main:
 	@cd $(TF_MAIN_DIR) && $(TF) apply --auto-approve
 
-destroy_main:
+dest_main:
```

Comment:
nit: I would keep this as `destroy_main`. `dest` is confusing and probably not worth it just to save 3 letters.